sdk/src/types/block/output/feature/issuer.rs

```diff
@@ -26,15 +26,55 @@ impl IssuerFeature {
     }
 }
 
-pub(crate) mod dto {
+mod dto {
+    use alloc::format;
+
     use serde::{Deserialize, Serialize};
 
-    use crate::types::block::address::dto::AddressDto;
+    use super::*;
 
-    #[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
-    pub struct IssuerFeatureDto {
+    #[derive(Serialize, Deserialize)]
+    struct IssuerFeatureDto {
         #[serde(rename = "type")]
-        pub kind: u8,
-        pub address: AddressDto,
+        kind: u8,
+        address: Address,
+    }
+
+    impl From<&IssuerFeature> for IssuerFeatureDto {
+        fn from(value: &IssuerFeature) -> Self {
+            Self {
+                kind: IssuerFeature::KIND,
+                address: value.0,
+            }
+        }
+    }
+
+    impl From<IssuerFeatureDto> for IssuerFeature {
+        fn from(value: IssuerFeatureDto) -> Self {
+            Self(value.address)
+        }
+    }
+
+    impl<'de> Deserialize<'de> for IssuerFeature {
```

Comment:
this happens a few times, can we derive the code? or prefer to keep it manual for each for readability? 
(For example that `serde_derive_impl!()`

Reply:
I suppose I could